### DISCUSSION TITLE
Main Discussion Thread

### DISCUSSION THREAD

**Discussion by killshot13** [killshot13/express-smtp-mailer#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scooper4000** [comment]
comment creer un mailer via webhost ?? si possible envoyer moi le php

**Reply by killshot13** [reply]
Cherchez-vous une version de ce code, mais écrite en php?

**Comment by faisalone** [comment]
As a newer to node.js I'm not sure how do I apply it. How do I connect it to my own API. Can you please give me some idea about it? Thanks in Advance.

**Reply by killshot13** [reply]
What are you using for your front-end? And is that where your API is coming from? Let's say, from a contact form or something like that?

**Reply by faisalone** [reply]
I just want to send some random emails from my front-end. It could be like a smtp server. It could be a contact from. I just want to see the implementation of the application. Please share a live demo of you.

I'm here to check you, you can connect me to you by meet or zoom if you want.

**Reply by faisalone** [reply]
If I create a contact from then how do I connect it?

**Reply by killshot13** [reply]
See if this makes sense. Here is an example of a contact form; it is a React component for the frontend that uses `axios` to transfer the submitted data to the Express app. 

![Web capture_1-9-2021_940_wsl localhost](https://user-images.githubusercontent.com/47695475/131686025-7ee5f1c5-d298-4aec-9f3c-3f4504aa39ec.jpeg)

In the development environment, the React app (frontend) would be running at http://localhost:3000 and you run the `express-smtp-mail-server` (backend) at http://localhost:5000 and they will talk to each other. No extra configuration is needed.

In production, this little line of code is what will symlink the frontend and backend permanently, preserving the communication routes you have created. This is already added for you in `/routes/routes.js` near the bottom.

![express-api](https://user-images.githubusercontent.com/47695475/131687188-128f46f5-9cba-47ca-ab90-8b383e628ce8.png)
